specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/bigDataPool.json

```diff
@@ -446,7 +446,7 @@
           "description": "List of custom libraries/packages associated with the spark pool.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/LibraryInfo"
+            "$ref": "../../../../data-plane/Microsoft.Synapse/preview/2019-06-01-preview/library.json#/definitions/LibraryResourceProperties"
```

Comment:
Can you not refer to the preview version from within a stable version?

Reply:
@amoghnatu : I had a similar doubt but a stable version for the data-plane is not created. Please provide your inputs.

Reply:
I have changed it to use LibraryInfo.